src/uct/ib/base/ib_iface.c

```diff
@@ -1594,8 +1597,10 @@ ucs_status_t uct_ib_iface_query(uct_ib_iface_t *iface, size_t xport_hdr_len,
     iface_attr->latency.c += numa_latency;
     iface_attr->latency.m  = 0;
 
-    /* Wire speed calculation: Width * SignalRate * Encoding */
-    wire_speed = (width * signal_rate * encoding) / 8.0;
+    /* Wire speed calculation: Width * SignalRate * Encoding * Num_paths */
+    roce_lag_level = uct_ib_iface_is_roce(iface) ?
```

Comment:
minor: rename "roce_lag_level" to "lag_level"

Reply:
isn't lag only ethernet specific? i'd keep `roce` part then

Reply:
it kinda weird that this var is used also for ib (set to 1) but it's relevant only for roce
maybe it should be num_paths?

Reply:
ok